lua/ruby_nvim/cmd.lua

```diff
@@ -89,4 +83,31 @@ M.test = function(current_line)
   run_in_floating_window(opts.load("test_cmd"), args)
 end
 
+-- :RubyBrowseGem
+
+M.browse_gem = function()
+  local line = api.nvim_get_current_line()
+  local gem_word = 'gem'
+  local gem_index = line:find(gem_word)
```

Comment:
Why do we need `gem_word` variable? Should we remove this if it's not used elsewhere?

```suggestion
  local gem_index = line:find('gem')
```

Or should we use it in other places instead of `'gem'`?